DUI3-DX/Converters/ArcGIS/Speckle.Converters.ArcGIS3/Layers/VectorLayerToSpeckleConverter.cs

```diff
@@ -61,12 +94,35 @@ public VectorLayer RawConvert(FeatureLayer target)
         // Same IDisposable issue appears to happen on Row class too. Docs say it should always be disposed of manually by the caller.
         using (Row row = rowCursor.Current)
         {
-          var element = _gisFeatureConverter.RawConvert(row);
+          GisFeature element = _gisFeatureConverter.RawConvert(row);
+
+          // replace "attributes", to remove non-visible layer attributes
+          Base elementAttributes = new();
+          foreach (FieldDescription field in fieldDescriptions)
+          {
+            elementAttributes[field.Name] = element.attributes[field.Name];
+          }
+          element.attributes = elementAttributes;
+
+          // differentiate between Ring Multipatches and Mesh Multipatches
+          if (
+            spekleGeometryType == "Multipatch"
+            && element.geometry != null
+            && element.displayValue != null
+            && element.geometry.Count > 0
+            && element.displayValue.Count == 0
```

Comment:
🤔 Why are we doing so many checks on Send using speckle objects instead of interrogating the GIS API?

I would think that the GIS feature converter would do this if it is needed, or am I missing something?

Reply:
killed this part. Indeed, too much verifications, but it still needs to be figured out. Multipatches are basically Meshes, but certain type of Multipatches have outer and inner rings, like polygons, so we cannot consistently convert all types multipatches to meshes. Needs some rethinking.

Reply:
And instead, if we turn them all into Polygons, then ArcGIS will be confused on receiving that, because it does not allow 3-dimentional polygons

Reply:
I don't think we can say that a `GIS Multipatch` is equivalent to a `Mesh`... They are drawn with 2d curves that represent closed areas, meshes are points grouped by faces.

The analogy between the 2 is computationally expensive and, if we cannot guarantee a consistent result, I would argue we should not be making them meshes at all.

Reply:
It has nothing to do with gis-gis. On the contrary, it tries to represent gis geometry in a cad friendly way. Also, the multipatches are not all curves with closed areas as I described above, just 1 type (Ring type), all the others are exactly like Speckle meshes: https://pro.arcgis.com/en/pro-app/latest/sdk/api-reference/topic27403.html
Btw, this part of code is already gone, I added instead GisMultipatch class (for multipatches that act like meshes) and GisPolygon3d (for multipatches that are enclosed areas, which is basically a polygon but not necessarily on a horizontal surface).